packages/web-app-files/src/search/filter/preview.ts

```diff
@@ -19,7 +19,13 @@ export default class Preview implements SearchPreview {
     // no cache required, the filtering is client only and fast enough to recalculate the set
     // of results every time on the fly
     const resources: any[] = filterResources(this.store.getters['Files/files'], term, 5)
-    const searchResult = resources.map((resource) => ({ id: resource.id, data: { ...resource } }))
+    let searchResult = resources.map((resource) => ({ id: resource.id, data: { ...resource } }))
```

Comment:
I would prefer to have `areHiddenFilesShown` and `searchResult` in one op, a reducer could do the job.  
Just my personal opinion, its optional!